config/crds/bundle_v1alpha1_componentset.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
```

Comment:
Does this need to be explicitly set?

Reply:
It's a result of the generation library. I'm guessing creationTimestamp doesn't have an omitempty tag